meson.build

```diff
@@ -18,6 +18,13 @@ namespace_name = 'Eknr'
 
 mathjax_dir = get_option('mathjax_dir')
 
+jasmine_report_argument = ''
```

Comment:
No need to add this stuff right now, I think. The integration with Jenkins isn't currently working, and I don't think anyone will have time to fix it in the near future.